repository/kafka/operator/templates/service-monitor.yaml

```diff
@@ -1,4 +1,6 @@
 {{ if eq .Params.ADD_SERVICE_MONITOR "true" }}
+# service-monitor cannot use toggle task
+# as KUDO Kafka cannot guarantee the third-party CRDs to be present
```

Comment:
This is because in case of a toggle task that is disabled, we'd try to remove an unknown resource type?

Reply:
right! so the plan will be in FATAL error always no matter if it's true or false as we cannot delete/create unknown resource type